.github/workflows/doxygen.yml

```diff
@@ -0,0 +1,46 @@
+name: Doxygen
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  CI_Doxygen:
```

Comment:
iirc jobs have a name too, so you could just name this `Build and publish documentation`